reports/aspace2alma/get_MARCxml.rb

```diff
@@ -4,6 +4,14 @@
 require 'net/sftp'
 require_relative '../../helper_methods.rb'
 
+#log to files
```

Comment:
Would it be easier to make a logger object? See https://ruby-doc.org/stdlib-3.1.0/libdoc/logger/rdoc/Logger.html#class-Logger-label-How+to+create+a+logger

Reply:
I looked at that and it seemed more involved, but if you think it'd be the better way to go, I can keep at it!